src/main/java/by/itacademy/javaenterprise/knyazev/servises/InformationService.java

```diff
@@ -0,0 +1,78 @@
+package by.itacademy.javaenterprise.knyazev.servises;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import by.itacademy.javaenterprise.knyazev.dao.TeachersDAO;
+import by.itacademy.javaenterprise.knyazev.dao.exceptions.TeacherExceptionDAO;
+import by.itacademy.javaenterprise.knyazev.entities.Teacher;
+
+public class InformationService {
+	private TeachersDAO teachersDAO;
+	private static final Logger logger = LoggerFactory.getLogger(InformationService.class);
```

Comment:
```suggestion
	private static final Logger logger = LoggerFactory.getLogger(InformationService.class);
	
	private TeachersDAO teachersDAO;

```
правильный порядок полей в классе